tests/rados/test_rados_preempt_scrub.py

```diff
@@ -85,7 +85,7 @@ def run(ceph_cluster, **kw):
 
             log_lines = [
                 "head preempted",
-                "WaitReplicas::react(const GotReplicas&) PREEMPTED",
+                "WaitReplicas::react(const GotReplicas&) PREEMPTED!",
```

Comment:
@SrinivasaBharath I request more clarity here.
The comparison happening in `def verify_preempt_log` is not == rather `in`
So previously if a subset of the sentence was not being found in log lines, how is adding an additional character '!' going to make any difference?

Reply:
I reverted the changes. The fix will be provided in a separate PR.